apps/laboratory/tests/shared/utils/project.ts

```diff
@@ -35,20 +35,21 @@ const braveOptions: UseOptions = {
 }
 
 const WAGMI_EMAIL_BASED_REGEX = /(?:smart-account\.spec\.ts|siwe-sa\.spec\.ts|social\.spec\.ts).*$/u
+const ETHERS_EMAIL_BASED_REGEX = /(?:social\.spec\.ts).*$/u
 
 const SOLANA_UNIMPLEMENTED_TESTS_REGEX =
-  /^(?!.*(?:email\.spec\.ts|siwe\.spec\.ts|canary\.spec\.ts|smart-account\.spec\.ts|social\.spec\.ts|siwe-sa\.spec\.ts|siwe-email\.spec\.ts)).*$/u
+  /^(?!.*(?:email\.spec\.ts|siwe\.spec\.ts|canary\.spec\.ts|smart-account\.spec\.ts|social\.spec\.ts|siwe-sa\.spec\.ts|siwe-email\.spec\.ts|wallet-features\.spec\.ts)).*$/u
 
 const customProjectProperties: CustomProjectProperties = {
   'Desktop Chrome/ethers': {
-    testIgnore: /(?:social\.spec\.ts).*$/u
+    testIgnore: ETHERS_EMAIL_BASED_REGEX
   },
   'Desktop Brave/ethers': {
-    testIgnore: /(?:email\.spec\.ts|smart-account\.spec\.ts|social\.spec\.ts).*$/u,
+    testIgnore: ETHERS_EMAIL_BASED_REGEX,
```

Comment:
@tomiir I guess the SA working on ethers right? We should be able to enable this I think

Reply:
Yessir, let's try and run it all 🔥